apps/judicial-system/web/src/routes/Shared/SignedVerdictOverview/SignedVerdictOverviewForm.tsx

```diff
@@ -0,0 +1,388 @@
+              value: workingCase.courtCaseNumber,
+            },
+            {
+              title: 'Embætti',
+              value: `${
+                workingCase.prosecutor?.institution?.name || 'Ekki skráð'
+              }`,
+            },
+            { title: 'Dómstóll', value: workingCase.court?.name },
+            { title: 'Ákærandi', value: workingCase.prosecutor?.name },
+            { title: 'Dómari', value: workingCase.judge?.name },
+          ]}
+          accusedName={workingCase.accusedName}
+          accusedNationalId={workingCase.accusedNationalId}
+          accusedAddress={workingCase.accusedAddress}
+          defender={{
+            name: workingCase.defenderName || '',
+            email: workingCase.defenderEmail,
+            phoneNumber: workingCase.defenderPhoneNumber,
+          }}
+        />
+      </Box>
+      {(workingCase.accusedAppealDecision === CaseAppealDecision.POSTPONE ||
+        workingCase.prosecutorAppealDecision === CaseAppealDecision.POSTPONE) &&
+        workingCase.rulingDate &&
+        (user?.role === UserRole.JUDGE ||
+          user?.role === UserRole.REGISTRAR) && (
+          <Box marginBottom={7}>
+            <AppealSection
+              rulingDate={workingCase.rulingDate}
+              accusedGender={
+                // Handle missing gender
+                workingCase.accusedGender ?? CaseGender.OTHER
+              }
+              accusedAppealDecision={workingCase.accusedAppealDecision}
+              prosecutorAppealDecision={workingCase.prosecutorAppealDecision}
+              accusedPostponedAppealDate={
+                workingCase.accusedPostponedAppealDate
+              }
+              prosecutorPostponedAppealDate={
+                workingCase.prosecutorPostponedAppealDate
+              }
+              setAccusedAppealDate={setAccusedAppealDate}
+              setProsecutorAppealDate={setProsecutorAppealDate}
+              withdrawAccusedAppealDate={withdrawAccusedAppealDate}
+              withdrawProsecutorAppealDate={withdrawProsecutorAppealDate}
+              isAppealDeadlineExpired={
+                workingCase.isAppealDeadlineExpired ?? false
+              }
+              isAppealGracePeriodExpired={
+                workingCase.isAppealGracePeriodExpired ?? false
+              }
+            />
+          </Box>
+        )}
+      <Box marginBottom={5}>
+        <Accordion>
+          <PoliceRequestAccordionItem workingCase={workingCase} />
+          <CourtRecordAccordionItem workingCase={workingCase} />
+          <RulingAccordionItem workingCase={workingCase} />
+          <AccordionItem
+            id="id_4"
+            label={`Rannsóknargögn (${
+              workingCase.files ? workingCase.files.length : 0
+            })`}
+            labelVariant="h3"
+          >
+            <CaseFileList
+              caseId={workingCase.id}
+              files={workingCase.files || []}
+              canOpenFiles={canCaseFilesBeOpened()}
```

Comment:
Prosecutors from shared prosecutors office should not see the open button.

Reply:
No prosecutor can open case files (see https://github.com/island-is/island.is/blob/2fecef2ffc197906224cc41197078bc2def82a96/apps/judicial-system/web/src/routes/Shared/SignedVerdictOverview/SignedVerdictOverviewForm.tsx#L137).

Reply:
I had a discussion with Baldur and Guðný about this last week. The decision was to block access for the prosecutors in the shared prosecutors office. This is now blocked server side with a 403 result. The conclusion from our discussion was that the open button would be removed for these prosecutors.

Note that prosecutors from the owning prosecutors office should still see the button and have access.